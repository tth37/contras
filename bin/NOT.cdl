@Head
NAME Not
INPUT a
OUTPUT out
SYMBOL NAND Nand1

@Body
Nand1(a = a, b = a) => (out = res)

@End
OUTPUT out = res

# .\contras.exe .\Not.cdl .\test\Not.in .\test\Not.out