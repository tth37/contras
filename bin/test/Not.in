@Case
a = 01
out = ?
@End