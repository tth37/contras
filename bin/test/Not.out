@Case
  a[0] = 01
out[0] = 10
